README.rst

```diff
@@ -37,32 +37,34 @@
 
 |CI|  |COV| |PEP| |DOC| |VERS| |PY| |FORUM|
 
-`PennyLane <https://pennylane.readthedocs.io>`_ is a cross-platform Python library for quantum machine learning,
-automatic differentiation, and optimization of hybrid quantum-classical computations.
+`PennyLane <https://pennylane.readthedocs.io>`_ is a cross-platform Python library for **differentiable quantum programming**. Train a quantum computer the same way as a neural network.
+ 
+.. raw:: html
 
-Learn more about quantum machine learning with PennyLane: view and download QML examples
-and demos over at https://pennylane.ai/qml.
+    <img src="https://raw.githubusercontent.com/XanaduAI/pennylane/master/doc/_static/jigsaw.png" width="600px"  align="center">
+    
+You can use PennyLane for quantum computing, quantum machine learning, quantum chemistry, and hybrid quantum-classical computations. Extensive examples, tutorials, and demos are available at https://pennylane.ai/qml.
```

Comment:
Suggest moving this paragraph above the graphic, it looks visually cleaner:
![image](https://user-images.githubusercontent.com/2959003/77226348-9757d080-6bc7-11ea-9a49-dc20b73cb5f6.png)